modules/checking-mco-status.adoc

```diff
@@ -0,0 +1,91 @@
+master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
+worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
+----
++
+Here there are three master and three worker nodes. All machines are updated
+and none are currently updating. Because none are degraded (False), you can
+tell that there are no problems with each applied `machineconfig`.
+
+. To see each existing `machineconfig`, type:
++
+[source,terminal]
+----
+$ oc get machineconfigs
+NAME                             GENERATEDBYCONTROLLER          IGNITIONVERSION  AGE
+00-master                        2c9371fbb673b97a6fe8b1c52...   3.1.0            5h18m
+00-worker                        2c9371fbb673b97a6fe8b1c52...   3.1.0            5h18m
+01-master-container-runtime      2c9371fbb673b97a6fe8b1c52...   3.1.0            5h18m
+01-master-kubelet                2c9371fbb673b97a6fe8b1c52…     3.1.0            5h18m
+...
+rendered-master-dde...           2c9371fbb673b97a6fe8b1c52...   3.1.0            5h18m
+rendered-worker-fde...           2c9371fbb673b97a6fe8b1c52...   3.1.0            5h18m
+----
+
+. You can view the contents of a particular machineconfig (in this case,
+`01-master-kubelet`). The trimmed output shows that this machineconfig
+contains both configuration files (`cloud.conf` and `kubelet.conf`) and a
+systemd service (Kubernetes Kubelet):
++
+[source,terminal]
+----
+$ oc describe machineconfigs 01-master-kubelet
+Name:         01-master-kubelet
+...
+Spec:
+  Config:
+    Ignition:
+      Version:  3.1.0
+    Storage:
+      Files:
+        Contents:
+          Source:   data:,
+        Mode:       420
+        Overwrite:  true
+        Path:       /etc/kubernetes/cloud.conf
+        Contents:
+          Source:   data:,kind%3A%20KubeletConfiguration%0AapiVersion%3A%20kubelet.config.k8s.io%2Fv1beta1%0Aauthentication%3A%0A%20%20x509%3A%0A%20%20%20%20clientCAFile%3A%20%2Fetc%2Fkubernetes%2Fkubelet-ca.crt%0A%20%20anonymous...
+        Mode:       420
+        Overwrite:  true
+        Path:       /etc/kubernetes/kubelet.conf
+    Systemd:
+      Units:
+        Contents:  [Unit]
+Description=Kubernetes Kubelet
+Wants=rpc-statd.service network-online.target crio.service
+After=network-online.target crio.service
+
+ExecStart=/usr/bin/hyperkube \
+    kubelet \
+      --config=/etc/kubernetes/kubelet.conf \ ...
+----
+
+. If something goes wrong with a machineconfig that you apply, you can always
+back out that change. For example, if you had a run `oc create -f ./myconfig.yaml`
+to apply a machineconfig, you could remove that machineconfig by typing:
++
+[source,terminal]
+----
+$ oc delete -f ./myconfig.yaml
+----
++
+If that was the only problem, your machines should return to a non-degraded state.
```

Comment:
"the nodes in the affected pool should return to..."

Reply:
Do we have anything that briefly mentions what a rendered config is, bc the pool is actually rolling back to the previous rendered config which seems worth mentioning